generate_brick.py

```diff
@@ -38,24 +39,28 @@ def make_tag_classes(G, tag):
     G.add((has_no_tag, OWL.complementOf, has_tag))
     return has_tag, has_no_tag
 
-#syntax for protege: http://protegeproject.github.io/protege/class-expression-syntax/
+
+# syntax for protege:
+# http://protegeproject.github.io/protege/class-expression-syntax/
 def add_restriction(klass, definition):
-    l = []
+    elements = []
     equivalent_class = BNode()
     list_name = BNode()
     for idnum, item in enumerate(definition):
         restriction = BNode()
-        l.append(restriction)
-        G.add( (restriction, A, OWL.Restriction) )
-        G.add( (restriction, OWL.onProperty, item[0]) )
-        G.add( (restriction, OWL.hasValue, item[1]) )
-    G.add( (BRICK[klass], OWL.equivalentClass, equivalent_class) )
-    G.add( (equivalent_class, OWL.intersectionOf, list_name) )
-    c = Collection(G, list_name, l)
+        elements.append(restriction)
```

Comment:
Where do these changes come from? It seems unrelated to the version changes. I am fine with out-of-scope changes if they are intended, but I just want to make sure if these are not breaking.

Reply:
Ah, I think it's related to the code style. I am fine with this.

Reply:
This might be an artifact of the merge from `master` in the aftermath of merging #94. Sorry the merge was messy. The tests pass though!